### PR TITLE
Paper experiments

### PR DESCRIPTION
**Pull request by davidahirshberg**
This branch has updated default parameters matching those we use in the paper, and furthermore includes a vignette, paper-results.Rmd, that outputs the tables and figures from the paper.

This vignette is not currently included in the specification for generating the docs site. It’d be good to see it included there. Because the simulations are long-running, if we want to build this vignette as part of the deploy process on azure, it probably makes sense to tweak the vignette to look for the file all-simulations.rds and if present, load results from it and forgo running the simulations in the chunk named run-simulations and the two chunks following that process its output. That way, we can put the file on azure so doc generation during deploy is fast.  At the moment, that .rds file is in the project Dropbox in the directory package-simulations.
This vignette uses some packages that aren’t used elsewhere —- see here: https://github.com/synth-inference/synthdid/blob/paper-experiments/vignettes/paper-results-details/session-info

It looks like conflicts should be resolved by taking the code from the branch. If we do that, there will be a failed test in test_against_reference because we don’t have the package cvxr that we need to run the reference implementation on azure.  If that’s the only failed test —- and it should be —- we can override and merge.

### CODE REVIEW

File: R/vcov.R [hunk 1]
```diff
@@ -23,11 +22,107 @@
...
+# if weights = NULL is passed explicitly, calculates the usual jackknife estimate of variance.
+# returns NA if there is one treated unit or, for the fixed-weights jackknife, one control with nonzero weight
+jackknife_se = function(estimate, weights = attr(estimate, 'weights')) { 
+    setup = attr(estimate, 'setup')
+    opts = attr(estimate, 'opts')
+    if (setup$N0 == nrow(setup$Y) - 1 || (!is.null(weights) && sum(weights$omega != 0) == 1)) { return(NA) }
+    theta = function(ind) {
+	weights.jk = weights
+	if (!is.null(weights)) { weights.jk$omega = sum_normalize(weights$omega[ind[ind <= setup$N0]]) }
+	estimate.jk = do.call(synthdid_estimate,
```
Comment (erikcs) [inline R/vcov.R:80]: there is a bug here, `opts` contains `update.omega = TRUE` and `update.lambda = TRUE`, causing the following line to recompute weights at every jackknife iteration

**Comment by davidahirshberg** [comment]
Or, because the slow bit is the bootstrap and placebo intervals, you could
just loop through on one computer and update the jackknife standard errors
only.

File: tests/testthat/test_synthdid.R [hunk 2]
```diff
@@ -1,7 +1,9 @@
 test_that("a simple workflow doesn't error", {
```
Comment (erikcs) [inline tests/testthat/test_synthdid.R:1]: The last PR ran package tests in [4 minutes](https://dev.azure.com/grf-labs/synth-inference/_build/results?buildId=782&view=logs&j=12f1170f-54f2-53f3-20dd-22fc7dff55f9).

In this PR they take 35 minutes...

Comment (erikcs) [reply tests/testthat/test_synthdid.R:1]: Your "paper-results.Rmd" builds on Azure https://synth-inference.github.io/synthdid/articles/paper-results.html, but the total build time now is close to 1 hr, which is the maximum Azure allows

**Review by erikcs — APPROVED** [review]

**Pull request merged**